### ISSUE TITLE
Add RECOFI codelists

### ISSUE THREAD

**Issue by eblondel** [fdiwg/fdi-codelists#28]

**Comment by jherfaut** [comment]
Files are not in the good folder, I don't know how to move them

**Comment by jherfaut** [comment]
Yes, I saw that Git is already install on Mac. I've download gitkraken, work in progress. But it is strange I try to delete the file (from the browser) and it is still appears

**Comment by eblondel** [comment]
If you open a file on the browser, you will see on the top-right corner a button "...". You can click on it and select "Delete file" on the bottom of the menu.

![image](https://github.com/fdiwg/fdi-codelists/assets/1324396/c268da41-51ef-465c-8bfa-a0bf692785d4)

**Comment by eblondel** [comment]
Hi @jherfaut have you been able to install git to manage properly these files?

**Comment by eblondel** [comment]
Hi @jherfaut hope you are going well, any progress on this side?

**Comment by eblondel** [comment]
@jherfaut any news?
